plugins/modules/network_settings_workflow_manager.py

```diff
@@ -1401,34 +1389,32 @@ def get_global_pool_cidr(self, global_pool_cidr, global_pool_name):
             self.status = "failed"
             return self.check_return_status()
 
-        is_pool_found = False
         value = 1
-        while (not is_pool_found):
+        while True:
             response = self.dnac._exec(
                 family="network_settings",
                 function="get_global_pool",
                 params={"offset": value}
             )
             value += 25
             if not isinstance(response, dict):
-                self.log("Failed to retrieve the global pool details - "
-                         "Response is not a dictionary", "CRITICAL")
                 self.msg = "Failed to retrieve the global pool details - Response is not a dictionary"
+                self.log(self.msg, "CRITICAL")
                 self.status = "failed"
                 return self.check_return_status()
 
             all_global_pool_details = response.get("response")
             if not all_global_pool_details:
-                self.msg = "The global_pool_name '{0}' is not valid under reserve_pool_details".format(global_pool_name)
+                self.log("Invalid global_pool_name '{0}' under reserve_pool_details".format(global_pool_name), "ERROR")
+                self.msg = "No global pool details is available with the global_pool_name: {0}".format(global_pool_name)
```

Comment:
self.msg = "There are no details available for the global pool named '{0}'".format(global_pool_name)
or 
self.msg = "No information found for the global pool named '{0}'".format(global_pool_name)